gems/pending/VolumeManager/LVM/scanner.rb

```diff
@@ -149,8 +150,10 @@ def self.readRaw(d, rlh)
     d.seek(osp, IO::SEEK_SET)
     da
   end # def self.readRaw
+  private_class_method :readRaw
 
   def self.readStruct(d, struct)
     OpenStruct.new(struct.decode(d.read(struct.size)))
   end # def self.readStruct
+  private_class_method :readStruct
```

Comment:
@jrafanie  @AndyStabler Looks good. I verified the affected methods in this class are just called internally, applied the changes to this module and successfully ran a smart state locally.

:+1: